.azure-pipelines/windows.sh

```diff
@@ -25,11 +35,29 @@ install_host_dmd() {
 # REASON: the preinstalled version is buggy (see also: https://github.com/dlang/dmd/pull/9398#issuecomment-468773638)
 install_grep
 
+if [ "$HOST_DMD_VERSION" == "LATEST" ]; then
+    download "http://downloads.dlang.org/releases/LATEST" LATEST
+    HOST_DMD_VERSION=`cat LATEST`
```

Comment:
```suggestion
    HOST_DMD_VERSION="$(cat LATEST)"
```

Backticks are deprecated in bash.

Reply:
The quotes should not be part of the variable. Are they removed?

Reply:
replaced backticks with $(), but left out the quotes

Reply:
Yes quotes will be removed. They are required in bash to signify the start and end of a variable as otherwise it would stop at the first whitespace. Not really an issue with the version number, but for the path names below.